### PR TITLE
External resources

### PR DESCRIPTION
**Pull request by sean-pwd**
Add youtube playlist to external resources page.

### CODE REVIEW

File: content/en/getting-started/external-learning-resources/index.md [hunk 1]
```diff
@@ -35,3 +35,7 @@ Hugo in Action is a step-by-step guide to using Hugo to create static websites.
 ### Video Playlist by Mike Dane
 
 Mike Dane explains the various features of Hugo via dedicated tutorials on [YouTube](https://www.youtube.com/watch?list=PLLAZ4kZ9dFpOnyRlyS-liKL5ReHDcj4G3&v=qtIqKaDlqXo).
+
+### Video Playlist by Sean Emerson of Future Web Design
```
Comment (jmooring) [inline content/en/getting-started/external-learning-resources/index.md:39]:
```suggestion
### Video Playlist by Sean Emerson
```

File: content/en/tools/other.md [hunk 2]
```diff
@@ -27,3 +27,4 @@ And for all the other small things around Hugo:
 * [Hugo SFTP Upload](https://github.com/thomasmey/HugoSftpUpload) Syncs the local build of your Hugo website with your remote webserver via SFTP.
 * [Emacs Easy Hugo](https://github.com/masasam/emacs-easy-hugo) Emacs package for writing blog posts in markdown or org-mode and building your site with Hugo.
 * [JAMStack Themes](https://jamstackthemes.dev/ssg/hugo/). JAMStack themes is a collection of site themes filterable by static site generator and supported CMS to help build CMS-connected sites using Hugo (linking to Hugo-specific themes).
+* [hugo-svg-font](https://github.com/future-wd/hugo-svg-font) A partial & shortcode for using SVG files just like svg fonts, but without the CSS overhead.
```
Comment (jmooring) [inline content/en/tools/other.md:30]:
```suggestion
```

**Review by jmooring — CHANGES_REQUESTED** [review]
The addition of the video playlist is welcome.

The addition of hugo-svg-font was rejected by the project lead in https://github.com/gohugoio/hugoDocs/pull/1567/.

**Review by coliff — APPROVED** [review]

**Comment by jmooring** [comment]
Requested changes not made. Closing.

**Pull request closed without merging**